magefile.go

```diff
@@ -0,0 +1,18 @@
+//go:build mage
+
+package main
+
+import (
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+)
+
+var env = map[string]string{"GOPRIVATE": "github.com/mesosphere"}
+
+type (
+	Test mg.Namespace
+)
+
+func (Test) Go() error {
+	return sh.RunWithV(env, "go", "test", "-v", "-race", "-coverprofile", "coverage.txt", "-covermode", "atomic", "./...")
+}
```

Comment:
[nice to have] We could dog food repo here using `dagger` that would be awesome

Reply:
Let me do it in a separate PR 🆗